packages/inquirer_oc/inquirer_oc.1.0.8/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A collection of common interactive command line user interfaces"
+description:
+  "A collection of common interactive command line user interfaces"
+maintainer: ["Ou Fumio"]
+authors: ["Ou Fumio"]
+license: "MIT"
```

Comment:
You seem to be missing LICENSE information in your repository. Ideally you should have either a LICENSE file or some license headers in the source files.